airbyte-cdk/python/airbyte_cdk/logger.py

```diff
@@ -22,17 +22,60 @@
 # SOFTWARE.
 #
 
-
+import logging
+import sys
 import traceback
 
 from airbyte_cdk.models import AirbyteLogMessage, AirbyteMessage
 
+TRACE_LEVEL_NUM = 5
+
+
+def get_logger():
+    logging.setLoggerClass(AirbyteNativeLogger)
+    logging.addLevelName(TRACE_LEVEL_NUM, "TRACE")
+    logger = logging.getLogger("airbyte.source.zabbix")
```

Comment:
again, why did you hardcode this name?

Reply:
also, you need to set up this for root-level logger probably. We are not particularly interested in return result, this will configure global object, so all loggers derived from root will follow this configuration, i.e.
```
logging.info(...)
```
will work as intended